x-pack/plugins/alerting/common/alert_schema/field_maps/mapping_from_field_map.ts

```diff
@@ -27,25 +27,8 @@ export function mappingFromFieldMap(
   });
 
   fields.forEach((field) => {
```

Comment:
This removes the `multi_field` property from the component templates. I didn't feel like they were necessary.